hash/a2600.xml

```diff
@@ -16935,7 +16935,7 @@ X-1414 : Streetracer
 	</software>
 
 	<software name="tunnelrnp" cloneof="tunnelrn">
-		<description>Tunnel Runner  (Prototype)</description>
+		<description>Tunnel Runner  (prototype)</description>
```

Comment:
There’s a superfluous space in this description – may as well fix it while you’re changing it.

Reply:
Removed the superfluous space. Thanks!